opm/output/eclipse/Summary.cpp

```diff
@@ -548,21 +546,21 @@ inline double group_keywords( E keyword,
 
     switch( keyword ) {
         /* Production rates */
-        case E::GWPR: return rate( rt::wat );
-        case E::GOPR: return rate( rt::oil );
-        case E::GGPR: return rate( rt::gas );
-        case E::GLPR: return rate( rt::wat ) + rate( rt::oil );
+        case E::GWPR: return - rate( rt::wat );
+        case E::GOPR: return - rate( rt::oil );
+        case E::GGPR: return - rate( rt::gas );
+        case E::GLPR: return (- rate( rt::wat )) + (- rate( rt::oil ));
 
         /* Production totals */
-        case E::GWPT: return accu + vol( rt::wat );
-        case E::GOPT: return accu + vol( rt::oil );
-        case E::GGPT: return accu + vol( rt::gas );
+        case E::GWPT: return accu + ( - vol( rt::wat ) );
+        case E::GOPT: return accu + ( - vol( rt::oil ) );
+        case E::GGPT: return accu + ( - vol( rt::gas ) );
```

Comment:
Sure. I did it this way to emphasise that accumulation is increasing and the result is positive, but I don't mind doing it the other way (or maybe later do an abs()).